doc/nrf/libraries/bluetooth_services/services/ddfs.rst

```diff
@@ -0,0 +1,61 @@
+.. _ddfs_readme:
+
+Direction and Distance Finding Service (DDFS)
+#############################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Bluetooth® LE GATT Direction and Distance Finding Service is a custom service that provides publication of distance, azimuth and elavation measurement data.
+Additional support for measurement configuration.
```

Comment:
This is a bit vague. What do you want to say actually? Does this service provide some support for Distance Measurement or ...?

Reply:
I wanted to say that the service allows publishing of measurements: distance, azimuth and elevation.
And additionally that it is possible to configure the parameters.